integration_tests/snaps/simple-waf/program/wscript_build

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python
+
+def write_header(tsk):
+	tsk.outputs[0].write('int abc = 423;\n')
+bld(rule=write_header, target='b.h', ext_out=['.h'])
+
+tg = bld.program(
+	features = 'aaa',
+	source   = 'main.c',
+	includes = '. ..',
+	#cflags   = ['-O3'], # for example
+	defines  = ['foo=bar'],
+	target   = 'myprogram',
+	use      = 'M')
+
+# just for fun, make main.c depend on wscript_build
+bld.add_manual_dependency('main.c', bld.path.find_resource('wscript_build'))
+
+# ----------------------------------------
+
+from waflib import TaskGen
```

Comment:
This as well as the other is not written from scratch - as I wrote in the commit the whole simple-waf program is from upstream waf demo section as-is.
I surely can move it up, but I'd rather prefer it pristine as provided by the waf project. That lessens the likelihood of somebody wondering why it is different than upstream and will easen if we ever have to update it as any upstream diff will apply